StRoot/StFwdTrackMaker/StFwdTrackMaker.cxx

```diff
@@ -455,7 +458,17 @@ int StFwdTrackMaker::Init() {
 
     mForwardData = std::shared_ptr<FwdDataSource>(new FwdDataSource());
     mForwardTracker->setData(mForwardData);
-    mForwardTracker->initialize( geoCache, mGenHistograms );
+    mForwardTracker->initialize( mGeoCache, mGenHistograms );
+
+    // geometry should be available from here (mForwardTracker will initialize cache if needed)
```

Comment:
I'm slightly concerned about pulling the z-locations from the geometry _and_ having fallback values hardcoded here.  

Is there at least logging in place to keep track of when / if we fall back to the default?